doc/developer-notes.md

```diff
@@ -839,6 +840,37 @@ namespace {
 #endif // BITCOIN_FOO_BAR_H
 ```
 
+Construct On First Use
+------------------------
+
+It is not possible to control the order in which non-local static objects in *different* translation units are initialized.
+This is the so-called "static initialization order problem". It could cause the use of an object:
+- before it is constructed (on program startup)
+- after it is destructed (at program exit)
+
+Both cases lead to undefined behavior.
+
+To prevent the static initialization order problem, use the Construct On First Use:
```

Comment:
s/use the/be sure to/

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/18881#issuecomment-624053939).